### PR TITLE
Create load balancer VMs in a thread.

### PR DESCRIPTION
**Pull request by tmarballi**

### CODE REVIEW

File: appscale/tools/agents/azure_agent.py [hunk 1]
```diff
@@ -332,6 +333,28 @@ def run_instances(self, count, parameters, security_configured, public_ip_needed
 
     return instance_ids, public_ips, private_ips
 
+  def setup_virtual_machine_creation(self, credentials, network_client,
+                                     parameters, subnet):
+    """ Sets up the network interface and creates the virtual machines needed
+    with the load balancer roles.
+    Args:
+        credentials: A ServicePrincipalCredentials instance, that can be used to
```
Comment (menivaitsi) [inline appscale/tools/agents/azure_agent.py]: Please use 2-space indentation here.

**Comment by tmarballi** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/2726/

**Comment by tmarballi** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/2765/

**Comment by tmarballi** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/2775/

**Pull request merged**